demos/link-prediction/random-walks/ctdne-link-prediction.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<p>Is it worth adding some text to give a bit more context about what we're looking at here? At least something like</p><p> </p><blockquote>As with the nodes, we can visualise the embeddings for each link to see how the positive links (with class <code>1</code> ) are reasonably well clustered and similarly for the false links (with class <code>0</code> ).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1056/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>